EzMap.Api/Services/IdentityService.cs

```diff
@@ -20,4 +21,41 @@ public Guid GetUserId()
 
         return tempId;
     }
+
+    public async Task<List<string>> VerifyPassword(string inputtedPassword)
```

Comment:
I still don't get why we need this instead of specifying the messages in FluentValidation